### ISSUE TITLE
web support

### ISSUE THREAD

**Issue by richard457** [FilledStacks/boxtout#20]
1. With this repo, this is good way to fully test stacked if it works on web, would be nice if is tested here.

**Comment by FilledStacks** [comment]
Good Idea. But we don't even have a customer app yet 😓 we'll do that when we tackle the web version 🌐
